packages/react/src/components/dialog/Dialog.module.scss

```diff
@@ -3,24 +3,25 @@
 @value medium-up from "../../styles/breakpoints.css";
 
 .dialogVisibleBodyWithHiddenScrollbars {
-  height: 100vh !important;
-  overflow: hidden !important;
+    overflow: hidden !important;
 }
 
 .dialogContainer {
   --accent-line-color: var(--color-bus);
   --overlay-color: rgba(0, 0, 0, 0.3);
-
+  -webkit-overflow-scrolling: none;
```

Comment:
I guess possible values are auto and touch: 
https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariCSSRef/Articles/StandardCSSProperties.html
But the Safari browser also suggests "unset"

Reply:
Otherwise this is good to go 👍

Reply:
Well seems like it damn, good spotting 👍 I'll fix